Gruntfile.js

```diff
@@ -1,10 +1,11 @@
 module.exports = function( grunt ) {
-var path = require( "path" );
-require( "load-grunt-config" )( grunt, {
-	configPath: [
-		path.join( process.cwd(), "tasks/options" ),
-		path.join( process.cwd(), "tasks" )
-	],
-	init: true
-} );
+	grunt.loadNpmTasks("axe-grunt-webdriver");
+	var path = require( "path" );
+	require( "load-grunt-config" )( grunt, {
+		configPath: [
+			path.join( process.cwd(), "tasks/options" ),
+			path.join( process.cwd(), "tasks" )
+		],
+		init: true
+	} );
```

Comment:
Why the indentation change on all of this? Can you please outdent it back to where it was to follow our style guide.